contributions.json

```diff
@@ -311,6 +311,10 @@
 			"icon": "$(sparkle)",
 			"commandPalette": "gitlens:enabled && !gitlens:untrusted && gitlens:gk:organization:ai:enabled"
 		},
+		"gitlens.ai.mcp.install": {
+			"label": "Install GitKraken MCP Server",
+			"commandPalette": "gitlens:enabled && !gitlens:untrusted && gitlens:gk:organization:ai:enabled"
+		},
```

Comment:
Eventually we will want versions of this command to other usages (e.g. home, etc) so we can understand the usage and make sure `source` is correct